tests/mock_crust/churn.rs

```diff
@@ -218,22 +228,32 @@ fn request_during_churn_node_to_group() {
         let data_id = data.identifier();
         let message_id = MessageId::new();
 
+        let except_index = added_index.unwrap_or(nodes.len());
+        let quorum_before = quorum(nodes.iter()
+            .enumerate()
+            .filter(|&(i, n)|
+                if i == except_index {
+                    false
+                } else {
+                    n.routing_table().is_recipient(&dst.to_destination())
+                })
```

Comment:
`i != except_index && n.routing_table().is_recipient(&dst.to_destination())` would be more concise (also below).